requirements/requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Framework must-haves
 numpy>=1.16.4
 torch>=1.0.0
+Pillow==6.2.2
```

Comment:
for torchvision compatability, you need Pillow<7
we have already solve this issue in another PR

Reply:
I know, but without this line, I couldn't pass the CI

Reply:
please merge with master, we have already fixed this issue